src/lib/viewers/iframe/IFrameLoader.js

```diff
@@ -33,6 +33,12 @@ class IFrameLoader extends AssetLoader {
         // The IFrame viewer is disabled when the file is a Boxdicom file and the disableDicom viewer option is enabled
         if (disableDicom && isDicomFile) {
             disabledViewers.push('IFrame');
+
+            // Removes boxdicom as a supported extension
+            const iframeViewer = this.viewers[0].EXT;
```

Comment:
Can we simplify things a bit by breaking out the viewers into multiple entries? Maybe something like:


```
const VIEWERS = [
    {
        NAME: 'IFrame',
        CONSTRUCTOR: IFrameViewer,
        REP: ORIGINAL_REP_NAME,
        EXT: ['boxdicom'],
    },
    {
        NAME: 'IFrame',
        CONSTRUCTOR: IFrameViewer,
        REP: ORIGINAL_REP_NAME,
        EXT: ['boxnote'],
    },
];

...

if (disableDicom && isDicomFile) {
    this.viewers = this.viewers.filter(viewer => !viewer.EXT.includes('boxdicom'));
}
```